tests/test_inter_area_agent.py

```diff
@@ -183,8 +183,8 @@ def bid(self, price: float, energy: float, buyer: str, seller: str,
         bid = Bid(bid_id, price, energy, buyer, seller,
                   original_bid_price=original_bid_price,
                   buyer_origin=buyer_origin)
-        self._bids.append(deepcopy(bid))
-        self.forwarded_bid = deepcopy(bid)
```

Comment:
Please make sure that these are also only precautions and dont safe us from a memeroy leak. Thanks a lot!

Reply:
Checked it out, on the terminal memory does not seem to increase significantly with a large number of devices.